app/javascript/components/Skeleton/index.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+import { classNames } from "$app/utils/classNames";
```

Comment:
This is breaking because the file is named `classnames`, not `classNames`

Reply:
@EmCousin fixed it, was a mac case-insensitive filesystems issue